metricflow/model/semantics/semantic_model_lookup.py

```diff
@@ -39,26 +40,26 @@ class SemanticModelLookup(SemanticModelAccessor):
 
     def __init__(  # noqa: D
         self,
-        model: SemanticManifest,
+        model: PydanticSemanticManifest,
```

Comment:
Wait, really? This seems like it'll cause problems later, so I hope it'll get updated somewhere down the line.

Reply:
Addressed in top level comment.